bazel/README.md

```diff
@@ -54,17 +56,3 @@ and all of its dependencies, and does not require amending `.bazelrc`. This
 is the preferred way, since it also unpacks the resulting tarball.
 
 See `test_external/` for an example using [embind](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html).
-
-### Using --config=wasm
```

Comment:
I'm not sure what you mean by "backwards-compatibility". Just because we update the config here doesn't imply it propagates to the user. In particular, this only affects _developers_ of _this_ repo, not external users.